_posts/2016-02-08-terminal-commands.md

```diff
@@ -0,0 +1,233 @@
+# Importance of using the terminal?
+
+Under multiple platforms the Operating System has the GUIs (graphical user interfaces), where you can as a user point, click and drag on multiple components of the OS, which makes the user experience extremely easy and friendly. 
+
+The traditional Unix environment are based into CLI (command line interface), where you as a user have the necessity to type commands to tell the computer what to do. Believe it or not, that is faster and more powerful to handle the Operating System.
+
+So, with the review made before, please join us reading this post which has the purpose to give you an introduction to use the fantastic world of CLI (command line interface).
+
+## Basic Concepts related to the CLI:
+
+- **Console:** This is the system as a whole. This is both the command line as well as the output from previous commands.
+- **Command Line:** This is the actual line in a console where you type your command.
+- **Prompt:** This is the beginning of the command line. It usually provides some contextual information like who you are, where you are and other useful info. After the prompt is where you will be typing commands.
+- **Terminal:** This is the actual interface to the console. The program we use to interact with the console is actually a **“terminal emulator”**, providing us the experience of typing into an old school terminal from the convenience of our modern graphical operating system.
+
+## How to start/access the terminal:
+### Linux:
+- **Unity** 
+    - Dash -> Search for Terminal
+    - Dash -> More Apps -> 'See More Results' -> Terminal
+    - Dash -> More Apps -> Accessories -> Terminal
+    - Keyboard Shortcut: Ctrl + Alt + T
+- **GNOME**
+    - Applications menu -> Accessories -> Terminal.
+    - Keyboard Shortcut: Ctrl + Alt + T
+- **Xfce (Xubuntu)**
+    - Applications menu -> System -> Terminal.
+    - Keyboard Shortcut: Super + T
+    - Keyboard Shortcut: Ctrl + Alt + T
+- **KDE (Kubuntu)**
+    - KMenu -> System -> Terminal Program (Konsole).
+- **LXDE (Lubuntu)**
+    - Menu -> Accessories -> LXTerminal.
+    - Keyboard Shortcut: Ctrl + Alt + T
+- **OSx:**
+    - ⌘(Command) + Space, type “Terminal” and then hit “Enter”
+    
+## Let's get started:
+
+**ls**
+
+The command “ls” stands for (List Directory Contents), List the contents of the folder, be it file or folder, from which it runs.If you run ls without any additional parameters, the program will list the contents of the current directory in short form.
```

Comment:
@marticam : I think you are missing a space here: "runs.If", also maybe you can make "ls" cursive or quote it as you did before